graphics.h

```diff
@@ -0,0 +1,30 @@
+#ifndef GRAPHICS_H
+#define GRAPHICS_H
+
+#include <SDL2/SDL.h>
+#include <iostream>
+
+
+ //constants
+ constexpr int kWindowWidth = 640;
+ constexpr int kWindowHeight = 480;
+ constexpr char kGameTitle[] = "DBZ";
+
+ //holds RGB values of background
+ constexpr struct {
+  char red = 0x0;
+  char green = 0x0;
+  char blue = 0xFF;
+
+  //opaque SDL constant
+  char alpha = SDL_ALPHA_OPAQUE;
+
+}BackgroundColor;//background colors
+
+class Graphics {
+public:
+ //initialize SDL library
+ bool InitSdl(); 
```

Comment:
Let's make InitSdl private since it is only called by CreateWindow.

Give it a better description in the comment above it. What does the return value mean?